src/diffusers/image_processor.py

```diff
@@ -168,6 +168,20 @@ def postprocess(
         image,
         output_type: str = "pil",
     ):
+        if output_type not in ["latent", "pt", "np", "pil"]:
+            deprecation_message = (
+                f"the output_type {output_type} is outdated and has been set to `np`. Please make sure to set it to one of these instead: "
+                "`pil`, `np`, `pt`, `latent`"
+            )
+            deprecate("Unsupported output_type", "1.0.0", deprecation_message, standard_warn=False)
+            output_type = "np"
+
+        if output_type == "latent":
+            return image
+
+        if self.config.do_normalize:
+            image = (image / 2 + 0.5).clamp(0, 1)
+
         if isinstance(image, torch.Tensor) and output_type == "pt":
```

Comment:
```suggestion
        if output_type == "pt":
```

Reply:
We should not do a check here. The tensor has to be of type PyTorch (we should have raised an error before hand otherwise)